IDesign/IDesign.StepByStep/Resources/InstructionFeedback.resx

```diff
@@ -0,0 +1,141 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Wordt dit bestand nog gebruikt?

Reply:
in de toekomst is het wel de bedoeling (zie #127 )